src/BlockTypes/ProductTag.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Product tag block.
+ *
+ * @package WooCommerce/Blocks
+ */
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * ProductTag class.
+ */
+class ProductTag extends AbstractProductGrid {
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'product-tag';
+
+	/**
+	 * This function is not necessary in this block.
```

Comment:
I don't understand this comment. :confused:

Reply:
Copied from product category :p Ill improve both.